src/components/PreviewFrame.jsx

```diff
@@ -105,15 +103,38 @@ class PreviewFrame extends React.Component {
     );
 
     if (bowser.is('Safari')) {
-      line = 1;
+      oneIndexedSourceLine = 1;
+    }
+
+    let oneIndexedOriginalLine = oneIndexedSourceLine;
+    let {column} = error;
+    if (this.props.compiledProject.sourceMap) {
+      const {SourceMapConsumer} = await retryingFailedImports(
+        () => import(
+          /* webpackChunkName: "mainAsync" */
+          'source-map' // eslint-disable-line comma-dangle
+        ),
+      );
+
+      const smc = new SourceMapConsumer(this.props.compiledProject.sourceMap);
+      const result = smc.originalPositionFor({
+        line: oneIndexedSourceLine,
+        column: error.column,
+      });
+
+      if (result.line !== null && result.column !== null) {
+        oneIndexedOriginalLine = result.line;
+        ({column} = result);
+      }
     }
 
     this.props.onRuntimeError({
       reason: normalizedError.type,
       text: normalizedError.message,
       raw: normalizedError.message,
-      row: line - 1,
-      column: error.column,
+      // Convert back to zero indexed
```

Comment:
I think this is pretty self-explanatory given your descriptive variable naming

Reply:
Updated